### PR TITLE
Add "Go to Show" context button

### PR DESCRIPTION
**Pull request by bohmandan**
A feature I have wanted for a while, to be able to go from for example an episode you find in "popular", to open a list of other episodes for that show.

### CODE REVIEW

**Comment by linqcan** [comment]
Thank you for your contribution!

I like this idea and really liked how you added test cases to your code! 👍

However, I can not merge this at this point as I am about to redo the fetching of the popular section due to backend changes. Those changes will make some of the code in this PR redundant.

I propose we keep this PR open and you can adjust the code when I have merged the new changes for the backend?

**Comment by linqcan** [comment]
I have now merged the changes. Please have a look and see if you can update this PR to be compatible with the new code base.

Let me know if you need any help!

**Comment by bohmandan** [comment]
Ok, so now that you have removed `MODE_CLIPS` i did a small refactoring as well, since this is the only valid case to add the context menu button with this option right?
(`        if not folder and params["mode"] == self.MODE_VIDEO:`)

And I am not that fond of me adding a test of another library (`helper`) in the `testSvt.py` suite of tests, but then again, I thought the most important was to make sure it doesnt break so using real data was prioritized.
If you approve you approve 🙂

And thanks again for maintaining this add-on!

File: resources/lib/listing/common.py
Comment (linqcan) [inline resources/lib/listing/common.py]: Use constant MODE_PROGRAM

File: tests/testSvt.py
Comment (linqcan) [inline tests/testSvt.py]: I think you can **revert all changes to this file**. The added test is not needed and you have a bad merge causing dead tests being re-added.

**Review by linqcan — CHANGES_REQUESTED** [review]
Thanks for looking into this.

I have made some comments that needs to be resolved before this can be merged. Please have a look at them.

Overall it looks good, good job! 👍

File: tests/testSvt.py
Comment (bohmandan) [reply tests/testSvt.py]: Ok done!

File: resources/lib/listing/common.py
Comment (bohmandan) [reply resources/lib/listing/common.py]: Fixed!

**Comment by bohmandan** [comment]
Ok then, I think it is ready!

**Review by linqcan — CHANGES_REQUESTED** [review]
The code looks fine now, thanks!

However, before I can merge I need you to do 2 things in Git:

- [ ] Rebase this branch on our latest version of branch _krypton_. This is needed since there are conflicts. Make sure to solve those conflicts without adding anything except what we see in this PR now.
- [ ] Squash your commits to one commit. https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Comment by bohmandan** [comment]
What conflicts do you mean? I dont see any conflicts.

Squashing, no problem.

**Comment by linqcan** [comment]
It's a git thing. It can't do an automatic 3 way merge of testSvt.py but maybe that will solve it self with the squashing. Squash and we will see what the GitHub gui says 😊

**Comment by bohmandan** [comment]
Ahum, after a lot of messing around with Git... 😄 
What about now, looks fine?

**Comment by linqcan** [comment]
Yes! Excellent!

Do you want a mention in the release notes?

**Comment by bohmandan** [comment]
> Do you want a mention in the release notes?

It is just a minor contribution, but why not - would be fun 🙂

**Comment by linqcan** [comment]
I'll make sure to add it! 👍🏻

Thanks for your contribution!

**Comment by linqcan** [comment]
@bohmandan I had to make some changes for non episodes (movies etc). If you are interested this is the commit: 8c97b8a1a6adf4d50dc94eee787fb3e07ea717ca

**Comment by linqcan** [comment]
Version 5.1.3 (rolling out now) should contain this feature 🎉 

Sorry for the delay!

**Pull request merged**
